pkg/volume/rbd/rbd_util.go

```diff
@@ -405,29 +413,24 @@ func (util *RBDUtil) AttachDisk(b rbdMounter) (string, error) {
 		mon := util.kernelRBDMonitorsOpt(b.Mon)
 		glog.V(1).Infof("rbd: map mon %s", mon)
 
-		errList := []error{}
-		if nbdToolsFound {
-			output, err = execRbdMap(b, "rbd-nbd", mon)
-			if err != nil {
-				glog.Infof("rbd-nbd: map error %v, rbd-nbd output: %s", err, string(output))
-				glog.V(4).Info("will retry using rbd after rbd-nbd failure")
-				errList = append(errList, err)
-			} else {
-				devicePath, mapped = waitForPath(b.Pool, b.Image, 10 /*maxRetries*/, true /*useNbdDriver*/)
-			}
-		}
-		if !mapped {
-			_, err = b.exec.Run("modprobe", "rbd")
-			if err != nil {
-				glog.Warningf("rbd: failed to load rbd kernel module: %v", err)
+		output, err = execRbdMap(b, "rbd", mon)
+		if err != nil {
+			glog.V(1).Infof("rbd: map error %v, rbd output: %s", err, string(output))
```

Comment:
Given that this could be happening even when people want to use rbd-nbd, I think we should adjust the log level to 3 or 4.

Reply:
it can be v1 and move to 420.